src/Scene/Quadtree.js

```diff
@@ -12,14 +12,39 @@
  */
 define('Scene/Quadtree', [
     'Scene/Layer',
+    'Core/Commander/InterfaceCommander',
     'Core/Geographic/Quad',
     'Renderer/NodeMesh'
-], function(Layer, Quad, NodeMesh) {
-
+], function(Layer, InterfaceCommander, Quad, NodeMesh) {
+
+
+    function commandQueuePriorityFunction(cmd) {
```

Comment:
I would move this function to NodeProcess since the priority of a command depends more on the nature of the node than the structure in which it is.